Assets/Scripts/GameController.cs

```diff
@@ -26,6 +26,7 @@ public class GameController : MonoBehaviour
     [Header("CPU Settings")]
     [SerializeField] private CPUConfig _cpuConfig;
     [SerializeField] private CPUMode _defaultCPUMode;
+    [SerializeField] private SensitivityManager _sensitivityManager;

```

Comment:
CPU Settings下に置くと勘違いしやすい。
新しくヘッダーを作るか、CPU Settingsより上に書いた方がよさそう。

```suggestion
    [SerializeField] private SensitivityManager _sensitivityManager;
    [Header("CPU Settings")]
    [SerializeField] private CPUConfig _cpuConfig;
    [SerializeField] private CPUMode _defaultCPUMode;
```